tsl/src/continuous_aggs/refresh.c

```diff
@@ -334,15 +339,15 @@ continuous_agg_refresh(PG_FUNCTION_ARGS)
 	 */
 	LockRelationOid(catalog_get_table_id(catalog, CONTINUOUS_AGGS_INVALIDATION_THRESHOLD),
 					AccessExclusiveLock);
-	continuous_agg_invalidation_threshold_set(cagg->data.raw_hypertable_id, refresh_window.end);
-	invalidation_process_hypertable_log(cagg, &refresh_window);
+	continuous_agg_invalidation_threshold_set(cagg->data.raw_hypertable_id, refresh_window->end);
+	invalidation_process_hypertable_log(cagg, refresh_window);
 
 	/* Start a new transaction. Note that this invalidates previous memory
 	 * allocations (and locks). */
 	PopActiveSnapshot();
 	CommitTransactionCommand();
 	StartTransactionCommand();
-	cagg = ts_continuous_agg_find_by_relid(cagg_relid);
+	cagg = ts_continuous_agg_find_by_mat_hypertable_id(mat_id);
```

Comment:
cagg and cagg_ht was looked up already in the calling function, and cagg was passed as an argument to this function. Is it necessary to look up again? Is there a better way, e.g., to pass on both to the internal function so that we need not look them up again?